deploy/cr.yaml

```diff
@@ -27,6 +27,7 @@ spec:
 
   - name: rs0
     size: 3
+    serviceAccountName: percona-server-mongodb-operator-workload
```

Comment:
Why we can not use 'default' service account ? Why do we need to use 'percona-server-mongodb-operator-workload' ?

Reply:
Default service account has it's own preconfigured user privileges. Workload account a completely new SA with very particular privileges needed only for security-context customization

Reply:
I've checked it again and it seems that such an SA was added by mistake. It should be utilized only inside security-context-field